apps/service_providers/migrations/0003_migrate_experiments_to_new_model.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.2 on 2023-10-24 12:01
+
+from django.db import migrations
+
+
+def migrate_llm_providers(apps, schema_editor):
+    LlmProvider = apps.get_model("service_providers", "LlmProvider")
+    Experiment = apps.get_model("experiments", "Experiment")
+
+    for experiment in Experiment.objects.all():
+        experiment.llm_provider_new_id = LlmProvider.objects.filter(old_id=experiment.llm_provider_id).first().id
+        experiment.save()
+
+
+class Migration(migrations.Migration):
+    initial = True
+
+    dependencies = [
+        ("service_providers", "0002_copy_llm_providers"),
```

Comment:
Interesting that this migration doesn't depend on `0048_experiment_llm_provider_new.py` in experiments, since it requires the `llm_provider_new` to exist on experiments?